python/lsst/obs/base/script/convertGen2RepoToGen3.py

```diff
@@ -184,10 +189,11 @@ def convert(gen2root, gen3root, instrumentClass, calibFilterType,
         butlerConfig = gen3root
     butler = lsst.daf.butler.Butler(butlerConfig, run=f"raw/{instrument.getName()}")
     convertRepoTask = ConvertRepoTask(config=convertRepoConfig, butler3=butler)
+    rerunsArg = {x: [instrument.getName()] for x in reruns} if reruns is not None else []
```

Comment:
This is a dict but isn't the reruns arg meant to take a `list` of `Rerun`?

Reply:
Looks like that was a recent change induced in DM-21849? I'm currently trying to integrate those changes now.